api/tasks/runner.go

```diff
@@ -173,10 +175,17 @@ func (t *task) prepareRun() {
 		return
 	}
 
-	if err := t.updateRepository(); err != nil {
-		t.log("Failed updating repository: " + err.Error())
-		t.fail()
-		return
+	if strings.HasPrefix(t.repository.GitURL, gitURLFilePrefix) {
+		repositoryPath := strings.TrimPrefix(gitURLFilePrefix, t.repository.GitURL)
```

Comment:
Wrong way round, the prefix is the 2nd argument

https://pkg.go.dev/strings#TrimPrefix